### ISSUE TITLE
deep scaffolding ends in an error

### ISSUE THREAD

**Issue by CrystalMethod** [aquaproj/aqua#1696]
## aqua version

Please use the latest version.

```console
$ aqua -v
aqua version 1.34.1 (3f727b05e5ffc29035b53028f487992c574580bb)
$ aqua-registry -v
aqua-registry version 0.1.8 (cc4161b67fe7319447ccf3d4d818014eef21a4b7)
```

## Environment

* OS (Windows, Linux, macOS, etc): darwin
* CPU Architecture (amd64, arm64, etc): amd64

## How to reproduce

```console
$ aqua-registry scaffold --deep orf/gping
+ aqua gr --deep --out-testdata pkgs/orf/gping/pkg.yaml orf/gping > pkgs/orf/gping/registry.yaml
WARN[0000] parse a tag as semver                         aqua_version=1.34.1 env=darwin/amd64 error="Malformed version: gping-v1.8.0" program=aqua tag_name=gping-v1.8.0
WARN[0000] parse a tag as semver                         aqua_version=1.34.1 env=darwin/amd64 error="Malformed version: gping-v1.7.0" program=aqua tag_name=gping-v1.7.0
WARN[0000] parse a tag as semver                         aqua_version=1.34.1 env=darwin/amd64 error="Malformed version: gping-v1.6.3" program=aqua tag_name=gping-v1.6.3
WARN[0000] parse a tag as semver                         aqua_version=1.34.1 env=darwin/amd64 error="Malformed version: gping-v1.6.2" program=aqua tag_name=gping-v1.6.2
WARN[0000] parse a tag as semver                         aqua_version=1.34.1 env=darwin/amd64 error="Malformed version: gping-v1.6.1" program=aqua tag_name=gping-v1.6.1
WARN[0000] parse a tag as semver                         aqua_version=1.34.1 env=darwin/amd64 error="Malformed version: gping-v1.6.0" program=aqua tag_name=gping-v1.6.0
WARN[0000] parse a tag as semver                         aqua_version=1.34.1 env=darwin/amd64 error="Malformed version: gping-v1.5.1" program=aqua tag_name=gping-v1.5.1
WARN[0000] parse a tag as semver                         aqua_version=1.34.1 env=darwin/amd64 error="Malformed version: gping-v1.5.0" program=aqua tag_name=gping-v1.5.0
WARN[0000] parse a tag as semver                         aqua_version=1.34.1 env=darwin/amd64 error="Malformed version: gping-v1.4.0" program=aqua tag_name=gping-v1.4.0
WARN[0000] parse a tag as semver                         aqua_version=1.34.1 env=darwin/amd64 error="Malformed version: gping-v1.3.2" program=aqua tag_name=gping-v1.3.2
WARN[0000] parse a tag as semver                         aqua_version=1.34.1 env=darwin/amd64 error="Malformed version: gping-v1.3.1" program=aqua tag_name=gping-v1.3.1
WARN[0000] parse a tag as semver                         aqua_version=1.34.1 env=darwin/amd64 error="Malformed version: gping-v1.3.0" program=aqua tag_name=gping-v1.3.0
WARN[0000] parse a tag as semver                         aqua_version=1.34.1 env=darwin/amd64 error="Malformed version: gping-v1.2.7" program=aqua tag_name=gping-v1.2.7
WARN[0000] parse a tag as semver                         aqua_version=1.34.1 env=darwin/amd64 error="Malformed version: gping-v1.2.6" program=aqua tag_name=gping-v1.2.6
WARN[0000] parse a tag as semver                         aqua_version=1.34.1 env=darwin/amd64 error="Malformed version: gping-v1.2.5" program=aqua tag_name=gping-v1.2.5
WARN[0000] parse a tag as semver                         aqua_version=1.34.1 env=darwin/amd64 error="Malformed version: gping-v1.2.4" program=aqua tag_name=gping-v1.2.4
WARN[0000] parse a tag as semver                         aqua_version=1.34.1 env=darwin/amd64 error="Malformed version: gping-v1.2.3" program=aqua tag_name=gping-v1.2.3
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x28 pc=0x142f699]

goroutine 1 [running]:
github.com/hashicorp/go-version.(*Version).String(0x0)
        /home/runner/go/pkg/mod/github.com/hashicorp/go-version@v1.6.0/version.go:369 +0x39
github.com/hashicorp/go-version.(*Version).Compare(0x0, 0x0)
        /home/runner/go/pkg/mod/github.com/hashicorp/go-version@v1.6.0/version.go:116 +0x27
github.com/hashicorp/go-version.(*Version).GreaterThanOrEqual(...)
        /home/runner/go/pkg/mod/github.com/hashicorp/go-version@v1.6.0/version.go:303
github.com/aquaproj/aqua/pkg/controller/generate-registry.(*Controller).getPackageInfoWithVersionOverrides.func1(0xc00078a000?, 0xc0008808e0?)
        /home/runner/work/aqua/aqua/pkg/controller/generate-registry/version_overrides.go:77 +0x3f
sort.order2_func(...)
        /opt/hostedtoolcache/go/1.20.1/x64/src/sort/zsortfunc.go:299
sort.median_func({0xc000578460?, 0xc0008808e0?}, 0x7, 0xe, 0x15, 0xc0005781c8)
        /opt/hostedtoolcache/go/1.20.1/x64/src/sort/zsortfunc.go:308 +0x4a
sort.choosePivot_func({0xc000578460?, 0xc0008808e0?}, 0x50?, 0x28b7820?)
        /opt/hostedtoolcache/go/1.20.1/x64/src/sort/zsortfunc.go:284 +0x13a
sort.pdqsort_func({0xc000578460?, 0xc0008808e0?}, 0x4400108?, 0x18?, 0xc00078a000?)
        /opt/hostedtoolcache/go/1.20.1/x64/src/sort/zsortfunc.go:89 +0xdb
sort.Slice({0x267ce40?, 0xc000890b10?}, 0x1f?)
        /opt/hostedtoolcache/go/1.20.1/x64/src/sort/slice.go:26 +0xfa
github.com/aquaproj/aqua/pkg/controller/generate-registry.(*Controller).getPackageInfoWithVersionOverrides(0xc000939080?, {0x2cf6348, 0xc0002ee1c0}, 0x7ff7bfefe346?, {0x7ff7bfefe346, 0x9}, 0xc00026e000)
        /home/runner/work/aqua/aqua/pkg/controller/generate-registry/version_overrides.go:76 +0x47c
github.com/aquaproj/aqua/pkg/controller/generate-registry.(*Controller).getPackageInfo(0xc00075c5c0, {0x2cf6348, 0xc0002ee1c0}, 0x0?, {0x7ff7bfefe346?, 0x100db07?}, 0x1)
        /home/runner/work/aqua/aqua/pkg/controller/generate-registry/generate.go:115 +0x58b
github.com/aquaproj/aqua/pkg/controller/generate-registry.(*Controller).genRegistry(0xc00075c5c0, {0x2cf6348, 0xc0002ee1c0}, 0xc0005792e0, 0x119d6b6?, {0x7ff7bfefe346, 0x9})
        /home/runner/work/aqua/aqua/pkg/controller/generate-registry/generate.go:54 +0x6e
github.com/aquaproj/aqua/pkg/controller/generate-registry.(*Controller).GenerateRegistry(0xc0005792c8?, {0x2cf6348, 0xc0002ee1c0}, 0x11?, 0x2cdecc0?, {0xc000429130?, 0x1, 0x1?})
        /home/runner/work/aqua/aqua/pkg/controller/generate-registry/generate.go:46 +0xbe
github.com/aquaproj/aqua/pkg/cli.(*Runner).generateRegistryAction(0xc000882d20, 0xc00075c400)
        /home/runner/work/aqua/aqua/pkg/cli/generate_registry.go:108 +0x286
github.com/urfave/cli/v2.(*Command).Run(0xc00037b040, 0xc00075c400, {0xc0000be140, 0x5, 0x5})
        /home/runner/go/pkg/mod/github.com/urfave/cli/v2@v2.23.7/command.go:271 +0x9eb
github.com/urfave/cli/v2.(*Command).Run(0xc00037bcc0, 0xc00075c140, {0xc0000501e0, 0x6, 0x6})
        /home/runner/go/pkg/mod/github.com/urfave/cli/v2@v2.23.7/command.go:264 +0xc4d
github.com/urfave/cli/v2.(*App).RunContext(0xc0002685a0, {0x2cf6348?, 0xc0002ee1c0}, {0xc0000501e0, 0x6, 0x6})
        /home/runner/go/pkg/mod/github.com/urfave/cli/v2@v2.23.7/app.go:333 +0x665
github.com/aquaproj/aqua/pkg/cli.(*Runner).Run(0xc000882d20, {0x2cf6348, 0xc0002ee1c0}, {0xc0000501e0, 0x6, 0x6})
        /home/runner/work/aqua/aqua/pkg/cli/runner.go:141 +0x20a7
main.core(0xc00020c5b0, 0xc000880c60)
        /home/runner/work/aqua/aqua/cmd/aqua/main.go:56 +0x277
main.main()
        /home/runner/work/aqua/aqua/cmd/aqua/main.go:30 +0x55
FATA[0000] aqua failed                                   aqua_version=0.1.8 env=darwin/amd64 error="execute a command: exit status 2" program=aqua-registry
```

## works without the parameter --deep

```console
$ aqua-registry scaffold orf/gping
+ aqua gr orf/gping > pkgs/orf/gping/registry.yaml
Update registry.yaml
+ aqua g orf/gping >> aqua-local.yaml
+ aqua -c aqua-all.yaml g orf/gping >> pkgs/orf/gping/pkg.yaml
+ aqua i --test
```

**Comment by suzuki-shunsuke** [comment]
Please use the latest version `v1.34.2`.
The bug was fixed in v1.34.2.

- https://github.com/aquaproj/aqua/releases/tag/v1.34.2
- https://github.com/aquaproj/generate-registry-deep/actions/runs/4333288731

**Comment by CrystalMethod** [comment]
Wasn't aware of this new release. Thanks!
